src/components/ModalPayment/ModalConfirmation/ModalConfirmation.tsx

```diff
@@ -55,12 +58,15 @@ const ModalConfirmation = (props: Props) => {
     }
   };
 
+  const confirmHeroImage =
+    props.sellerId === 'light-up-chinatown' ? lucHeroImage : confirmationPic;
```

Comment:
Is there a 'light-up-chinatown' within a sellerId? LUC is a project id @jacoblb64?

Reply:
Not right now, see relevant [Trello Card](https://trello.com/c/ltqMmRzu). Should add a TODO comment tho

Reply:
Yeah, we need some bigger refactoring which we haven't done yet